examples/vision/horses_or_humans.exs

```diff
@@ -29,7 +33,7 @@ defmodule HorsesOrHumans do
 
   defnp augment(inp) do
     # Normalize
-    inp = inp / 255.0
+    inp = inp / Nx.Constants.max(@input_type)
```

Comment:
For me, this change makes it more confusing. The fact that the max value of the `u8` type is indeed 255 and the max value of `inp` is also 255 is rather an incidence. In my opinion, the comment that the maximum value is 255.0 so we normalize to [0,1] interval by dividing by 255.0 would be better.

Reply:
Right... good call!
Will update.

Reply:
I figured maybe something like this https://github.com/elixir-nx/axon/pull/512/commits/13f6662c2bf3d2b1eccef989de9f381ac2d391b6 could help in making it clear this is the maximum possible integer value of each image pixel channel.

Reply:
Yeap, this looks good :)